docs/dev/src/ci.md

```diff
@@ -19,3 +19,25 @@ To run `e2e-test` and `e2e-source-test` for `main-cron` in your pull request:
 1. Add `ci/run-e2e-test`.
 2. Add `ci/run-e2e-source-tests`.
 3. Add `ci/main-cron/run-selected` to skip all other steps which were not selected with `ci/run-xxx`.
+
+## Main Cron Bisect Guide
+
+1. Create a new build via buildkite: https://buildkite.com/risingwavelabs/main-cron/builds/#new
+2. Add the following environment variables:
+   - `START_COMMIT`: The lower bound (inclusive) of the regression range.
+   - `END_COMMIT`: The commit hash of the bad commit we observed.
+   - `BISECT_BRANCH`: The branch name where the bisect will be performed.
+   - `BISECT_STEPS`: The `CI_STEPS` to run during the bisect. Separate multiple steps with a comma.
+                     You can check the labels for this in `main-cron.yml`.
+   - `CI_STEPS`: Just put `disable-build` here, this is used to skip other steps we are not interested in.
```

Comment:
nit: why not simply put find-regression in a separate pipeline? So that we don't need additional `disable-build`

Reply:
True. Let me try that.

Reply:
+1. Don't quite understand what it's for. 😢

Reply:
Fixed: https://buildkite.com/risingwavelabs/main-cron-bisect/builds/5